Firmware/printer.cfg

```diff
@@ -0,0 +1,275 @@
+#if your fan will not start
+off_below: 0.13
+cycle_time: 0.010
+
+
+[idle_timeout]
+timeout: 1800
+
+
+[homing_override]
+axes: z
+set_position_z: 0
+gcode:
+   G90
+   G0 Z5 F600
+   G28 X Y
+   G0 X30 Y0 F3600 
+   
+   G28 Z
+   G0 Z10 F1800
+   G0 X60 Y60 Z20 F3600
+
+
+# Tool to help adjust bed leveling screws. One may define a
+# [bed_screws] config section to enable a BED_SCREWS_ADJUST g-code
+# command.
+[bed_screws]
+screw1: 60,5
+screw1_name: front screw
+screw2: 5,115
+screw2_name: back left
+screw3: 115,115
+screw3_name: back right
+
+
+[gcode_macro PRINT_START]
+#   Use PRINT_START for the slicer starting script - please customise for your slicer of choice
+gcode:
+    G28                            ; home all axes
+    G1 Z20 F3000                   ; move nozzle away from bed
+   
+[gcode_macro PRINT_END]
+#   Use PRINT_END for the slicer ending script - please customise for your slicer of choice
+gcode:
+
+    M400                           ; wait for buffer to clear
+    G92 E0                         ; zero the extruder
+    G1 E-4.0 F3600                 ; retract filament
+    G91                            ; relative positioning
+
+    #   Get Boundaries
+    {% set max_x = printer.configfile.config["stepper_x"]["position_max"]|float %}
+    {% set max_y = printer.configfile.config["stepper_y"]["position_max"]|float %}
+    {% set max_z = printer.configfile.config["stepper_z"]["position_max"]|float %}
+
+    #   Check end position to determine safe direction to move
+    {% if printer.toolhead.position.x < (max_x - 20) %}
+        {% set x_safe = 20.0 %}
+    {% else %}
+        {% set x_safe = -20.0 %}
+    {% endif %}
+
+    {% if printer.toolhead.position.y < (max_y - 20) %}
+        {% set y_safe = 20.0 %}
+    {% else %}
+        {% set y_safe = -20.0 %}
+    {% endif %}
+
+    {% if printer.toolhead.position.z < (max_z - 2) %}
+        {% set z_safe = 2.0 %}
+    {% else %}
+        {% set z_safe = max_z - printer.toolhead.position.z %}
+    {% endif %}
+
+    G0 Z{z_safe} F3600    ; move nozzle up
+    G0 X{x_safe} Y{y_safe} F20000    ; move nozzle to remove stringing
+    TURN_OFF_HEATERS
+    M107                           ; turn off fan
+    G90                            ; absolute positioning
+    G0 X60 Y{max_y} F3600          ; park nozzle at rear
+	
+[gcode_macro LOAD_FILAMENT]
+gcode:
+   M83                            ; set extruder to relative
+   G1 E280 F1800                  ; quickly load filament to down bowden
+   G1 E30 F300                    ; slower extrusion for hotend path
+   G1 E15 F150                    ; prime nozzle with filament
+   M82                            ; set extruder to absolute
+    
+[gcode_macro UNLOAD_FILAMENT]
+gcode:
+   M83                            ; set extruder to relative
+   G1 E10 F300                    ; extrude a little to soften tip
+   G1 E-380 F1800                 ; retract filament completely
+   M82                            ; set extruder to absolute
```

Comment:
The gearing ratio you specify is for mini ab, but here you have a long bowden config, this'll not go well

Reply:
Do  you have a better one because yes I've experienced a jam with this one